it/connectors/VatSubscriptionConnectorISpec.scala

```diff
@@ -50,7 +50,8 @@ class VatSubscriptionConnectorISpec extends IntegrationBaseSpec {
     lastName = Some("Taylor"),
     organisationName = Some("D Taylor's Cars"),
     tradingName = Some("DT Autos"),
-    contactPreference = Some(ContactPreference.digital)
+    contactPreference = Some(ContactPreference.digital),
+    changeIndicators = None
```

Comment:
I would put this as `Some` and update the JSON in `VatSubscriptionStub` to include the change indicators JSON as a way to test that it's being read correctly.